skore/src/skore/sklearn/_estimator/report.py

```diff
@@ -178,11 +188,16 @@ def clear_cache(self) -> None:
         ...     y_test=y_test,
         ... )
         >>> report.cache_predictions()
+        >>> assert len(report._cache) > 1  # cache is not empty
```

Comment:
Can you add the info in the long summary that we don't remove this key related to fit.